quantum/keymap_extras/keymap_norman.h

```diff
@@ -0,0 +1,89 @@
+
+/* Copyright 2019 Torben Hoffmann
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "keymap.h"
+// For software implementation of norman
+#define NM_Q    KC_Q
+#define NM_W    KC_W
+#define NM_D    KC_E
+#define NM_F    KC_R
+#define NM_K    KC_T
+#define NM_J    KC_Y
+#define NM_U    KC_U
+#define NM_R    KC_I
+#define NM_L    KC_O
+#define NM_SCLN KC_P
+
+#define NM_A    KC_A
+#define NM_S    KC_S
+#define NM_E    KC_D
+#define NM_T    KC_F
+#define NM_G    KC_G
+#define NM_Y    KC_H
+#define NM_N    KC_J
+#define NM_I    KC_K
+#define NM_O    KC_L
+#define NM_H    KC_SCLN
+#define NM_COLN LSFT(NM_SCLN)
```

Comment:
IMO Line 42 should be moved to Line 31, just below the definition for `NM_SCLN`.